nameserver/zone.go

```diff
@@ -18,6 +18,7 @@ var rdnsDomainLen = len(RDNSDomain) + 1
 
 type Zone interface {
 	AddRecord(ident string, name string, ip net.IP) error
+	DeleteRecords(ident *string, ip *net.IP, name *string) (count int)
```

Comment:
1) It's not clear to me that we should retain those other methods at all - we could make all clients use `DeleteRecords`
2) I'm not sure what the error could be? Failure to delete something at all?